src/amo/components/AddonReviewCard/index.js

```diff
@@ -437,18 +436,22 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
               review={review}
               byLine={!this.isRatingOnly() && byLine}
               showRating={!this.isReply() && showRating}
+              showDeveloperResponseHeading={
+                this.isReply() && !isReplyToReviewId
+              }
             />
-            {this.isRatingOnly() && (
-              <Button
-                className="AddonReviewCard-writeReviewButton"
-                onClick={this.onClickToEditReview}
-                href="#writeReview"
-                buttonType="action"
-                puffy
-              >
-                {i18n.gettext('Write a review')}
-              </Button>
-            )}
+            {this.isRatingOnly() &&
+              !isUserProfile && (
```

Comment:
Yes, by default it showed a full width button and that looked ridiculous, so I hid it. I'll try keeping a smaller button and see how that looks.

Reply:
I reduced the width of the button, but kept it `puffy` for now, because that's a simple css change, rather than trying to use a `micro` button instead (which I think might look better, although this seems like a more obvious call to action). Here's what it looks like now:

![screenshot 2018-09-19 14 11 36](https://user-images.githubusercontent.com/142755/45772517-f117bd00-bc15-11e8-9b79-e3c934b9bc1d.png)